lxc/image.go

```diff
@@ -939,6 +942,15 @@ func (c *cmdImageInfo) Run(cmd *cobra.Command, args []string) error {
 		fmt.Printf("    Alias: %s\n", info.UpdateSource.Alias)
 	}
 
+	if len(info.Profiles) == 0 {
+		fmt.Printf(i18n.G("Profiles: ") + "[]\n")
+	} else {
+		fmt.Println(i18n.G("Profiles:"))
```

Comment:
Those lines are causing an addition to the translation template, this then requires an additional commit which you can generate by doing:
 - make i18n
 - git commit -s -m "i18n: Update translation template" po/